deploy/Dockerfile

```diff
@@ -1,17 +1,19 @@
 # Image used for deployment. The Circle image is not used here as it does not
 # have an ARM build (M1 chip requires emulation).
-# Keep synchronized with ci/Dockerfile.circle_build
-FROM openjdk:11
+# Keep synchronized with ci/Dockerfile.circle_build and api/gradle.build
+FROM eclipse-temurin:17
```

Comment:
doesn't need to be this PR...wonder if we should use appsec approved images like these https://github.com/DataBiosphere/leonardo/blob/develop/Dockerfile#L18

Reply:
interesting... do you have documents about these images?

Reply:
Those are appsec "blessed" images
https://github.com/broadinstitute/dsp-appsec-blessed-images